Cargo.toml

```diff
@@ -0,0 +1,26 @@
+[package]
+name = "tree-sitter-hack"
+description = "hack grammar for the tree-sitter parsing library"
+version = "0.0.4"
+keywords = ["incremental", "parsing", "hack"]
+categories = ["parsing", "text-editors"]
+repository = "https://github.com/slackhq/tree-sitter-hack"
+edition = "2018"
+license = "MIT"
+
+build = "bindings/rust/build.rs"
+include = [
+  "bindings/rust/*",
+  "grammar.js",
+  "queries/*",
+  "src/*",
+]
+
+[lib]
+path = "bindings/rust/lib.rs"
+
+[dependencies]
+tree-sitter = "0.19.3"
```

Comment:
At first I was confused why this is 0.19.3 when we are upgrading to 0.19.4, but it is clear here: https://crates.io/crates/tree-sitter/versions. Leaving this comment for posterity.

Reply:
I was also confused by this but justified it to myself as the generator for this file knowing what it was doing and seeing other repos doing the same thing (using 0.19.3 in Cargo.toml and 0.19.5 in package.json), although none in the org github.com/tree-sitter.

Although, not sure what the link shows or how it makes this clear, think I'm missing something.

Reply:
The page suggested to me that rust bindings are their own release cycle. See https://github.com/tree-sitter/tree-sitter/tree/master/lib/binding_rust. And the version was updated for 0.19.5 in https://github.com/tree-sitter/tree-sitter/commit/aabe6100d0fc4b0758f517f32b115c9192c666d2#diff-76d93026086e2e1daba5002a8da69a5a78bb7447363b2349574ac1b65862033d to 0.19.3.